### ISSUE TITLE
Small time zones in Canada

### ISSUE THREAD

**Issue by ArcticGnome** [evansiroky/timezone-boundary-builder#90]
I've been working on trying to map the boundaries of IANA time zones in Canada in OSM. I'm glad to see that the most recent version of your map fixed some of the smaller time zones that used to be shown as limited to one city. However, I've noticed a couple more things that need to be fixed.

- America/Swift_Current is difficult to map exactly because it's dependent on 1970s school districts. But my research of southern part of the border is a bit different than yours. Also, there are some towns on Lake Athabasca that follow America/Swift_Current. You can either show this by making America/Swift_Current two blocks, or do what I did and draw a straight line through the mostly-uninhabited space between Lake Athabasca and the south. (https://www.openstreetmap.org/relation/10424167)

- America/Glace_Bay includes all of Cape Breton Island, plus some of the eastern mainland. I think it also includes some parts on the Bay of Fundy shore around Windsor, but I'm still working on finding the exact boundary. (https://www.openstreetmap.org/relation/10438273)

- The boundary between America/Toronto and America/Nipigon is strange. So far as I can tell from the Shanks Atlas and online searches, most of Ontario did not use DST in the early 1970s, and was thus in America/Nipigon. Only the big cities followed DST. In most cases, not even the towns _between_ cities used DST. But that turns America/Toronto into a messy patchwork. (https://www.openstreetmap.org/relation/6483867)

**Comment by evansiroky** [comment]
Hello @ArcticGnome! Thanks for all the research and OSM edits. I'll try to incorporate this into the next release. I'm curious, do you have a publicly available doc that lists the sources of your research? It would be good to cite that somehow.

**Comment by ArcticGnome** [comment]
Sorry for the late reply. My time zones in Yukon, Northwest Territories,
and Quebec all come from legislation, and are therefore exact. In Ontario
and New Nova Scotia I looked at when each town adopted DST according to
Thomas G Shanks "The International Atlas" 3rd ed, and then drew lines to
enclose those towns, following county boundaries if they were nearby. For
Saskatchewan, I also started with the Shanks atlas, but because time zones
there were based on what the closest school was at the time, I more often
drew the line at the midpoint between population centres.
